server/sandbox_network.go

```diff
@@ -81,17 +76,32 @@ func (s *Server) getSandboxIP(sb *sandbox.Sandbox) (string, error) {
 	}
 
 	podNetwork := newPodNetwork(sb)
-	result, err := s.netPlugin.GetPodNetworkStatus(podNetwork)
+	results, err := s.netPlugin.GetPodNetworkStatus(podNetwork)
 	if err != nil {
 		return "", fmt.Errorf("failed to get network status for pod sandbox %s(%s): %v", sb.Name(), sb.ID(), err)
 	}
+	return s.getSandboxIPsFromResult(results)
+}
 
-	res, err := cnicurrent.GetResult(result[0])
-	if err != nil {
-		return "", fmt.Errorf("failed to get network JSON for pod sandbox %s(%s): %v", sb.Name(), sb.ID(), err)
+func (s *Server) getSandboxIPsFromResult(results []cnitypes.Result) (string, error) {
```

Comment:
Are we guaranteed that we will only get one IP address? If so, how do we know from CNI config which one is the primary?

Reply:
I opted to fail if there was more than one IP found in a config, because it seemed like we can't support multiple.

Reply:
From this PRs perspective we should be picking the default network but I want to make sure we don't break multus cc: @dcbw